src/IdeHelperServiceProvider.php

```diff
@@ -37,6 +39,10 @@ class IdeHelperServiceProvider extends ServiceProvider
      */
     public function boot()
     {
+        if ($this->app['config']->get('ide-helper.listen_on_commands')) {
```

Comment:
the default for `get()` (2nd parameter) should be `false` for clarity sake

(it will work anyway because it's `null`, but…)